source/whats-new.txt

```diff
@@ -30,6 +32,20 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-5.2:
+
+What's New in 5.2
+-----------------
+
+New features of the 5.2 {+driver-short+} release include:
+
+- The driver now supports automatically obtaining Azure credentials when using
+  automatic Queryable Encryption
```

Comment:
S: maybe we can also link to https://www.mongodb.com/docs/v6.0/core/queryable-encryption/tutorials/azure/azure-automatic here like in the release notes

Reply:
also end with period

Reply:
Looking through that page, it doesn't look like we've actually documented this change there, so I think we should leave the link off for now